hackathon_site/event/jinja2/event/dashboard_base.html

```diff
@@ -32,14 +32,36 @@ <h2 class="formH2">Complete your application</h2>
             {% elif application is not none and review is not none %}
                 <br />
                 {% if review.status == "Accepted" %}
-                    <h4 class="formH2"><b>Congratulations!</b> You've been accepted into {{ hackathon_name }}!</h4>
-                    <p><strong>Status</strong>: {{ status }}</p>
+                    {% if application.rsvp == True %}
+                        <h4 class="formH2"><b>Thanks for RSVP'ing!</b> See you at {{ hackathon_name }}!</h4>
+                    {% elif application.rsvp == False %}
+                        <h4 class="formH2"><b>Thanks for letting us know.</b> Hope to see you next year at {{ hackathon_name }}</h4>
+                    {% else %}
+                        <h4 class="formH2"><b>Congratulations!</b> You've been accepted into {{ hackathon_name }}!</h4>
+                    {% endif %}
+                    <p style="display: flex;">
+                        <strong>Status:&nbsp;</strong>
+                        {% if status == "Cannot Attend (Declined)" %} 
+                            <span class="material-icons errorText" style="margin-right: 3px;">
+                                highlight_off
+                            </span>
+                        {% elif status == "Will Attend (Accepted)" %} 
+                            <span class="material-icons successText" style="margin-right: 3px;">
+                                check_circle_outline
+                            </span>
+                        {% endif %}
+                        {{ status }}
+                    </p>
                     <br>
                     {% if not rsvp_passed %}
-                        <p>To confirm your attendance at {{ hackathon_name }}, please RSVP using the accept or decline buttons below.</p> <br />
-                        <p>RSVP by <b>{{ rsvp_deadline }} (11:59PM EST)</b>, otherwise your spot will be given to other applicants.
-                            Alternatively you can decline the offer. You can change your response at any time before <b>{{ rsvp_deadline }} (11:59PM EST)</b>.
-                        </p> <br />
+                        {% if status == "Accepted, awaiting RSVP" %}
```

Comment:
This one can just become `if application.rsvp is None`. The full condition to generate this status string in the view is [here](https://github.com/ieeeuoft/hackathon-template/blob/develop/hackathon_site/event/views.py#L129). The first two conditions, that there is a review and that they were accepted, are checked above here in the template so we're good on them.